packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -267,17 +375,19 @@ const SideNav = ({
       <SideNavExpandContainer
         odysseyDesignTokens={odysseyDesignTokens}
         isSideNavCollapsed={isSideNavCollapsed}
+        expandedWidth={expandedWidth}
         data-se="expanded-region"
```

Comment:
I know we are trying to phase out `data-se` eventually. Do we need this here? If we do, do we want the consumer to pass this in?

Reply:
If this is soley for internal testing, we should probably use `testId` instead